example/assets/simple/text_2.svg

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" standalone="no"?>
 <!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" 
   "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
-<svg viewBox="0 0 1000 300"
+<svg viewBox="0 0 1000 350"
```

Comment:
Rather than modifying this test, could you just create a new SVG with the feature(s) you want? That way there's no confusion about whether this could have broken a previous golden.

Once you upload it, I can regenerate the goldens on Linux and upload them (or you can, if you have access to a Linux machine). I use Linux becuase it's generally available on CI...

Reply:
Got it! I will make it today